Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/ModalInfoView/ModalInfoViewModel.swift

```diff
@@ -7,3 +7,11 @@ public protocol ModalInfoViewModel {
     var title: GDSLocalisedString { get }
     var body: GDSLocalisedString { get }
 }
+
+/// View model for extra configuration on the `ModalInfoViewController`
+/// - `bodyTextColour` type is ``UIColor``
+/// - `preventModalDismiss` type is ``Bool``
+public protocol ModalInfoExtraViewModel {
+    var bodyTextColour: UIColor { get }
```

Comment:
`bodyTextColour` possibly fits better as part of the `ModalInfoViewModel`, and then we can default it to whatever the default colour is (presumably `UIColor.label`?)

Also, please can we be consistent with Swift and use the American spelling (I know, I know 😣)

Reply:
Cool cool, will do
Yeah I've never really known what to do with British vs American spelling in code but will follow that practice  from now on... time to adding Zs 😅